addons/mail/static/src/xml/activity.xml

```diff
@@ -18,7 +18,7 @@
                     <div class="o_thread_message_sidebar">
                         <div class="o_avatar_stack">
                             <img t-attf-src="/web/image#{activity.user_id[0] >= 0 ? ('/res.users/' + activity.user_id[0] + '/image_128') : ''}" class="o_thread_message_avatar rounded-circle mb8" t-att-title="activity.user_id[1]" t-att-alt="activity.user_id[1]"/>
-                            <i t-att-class="'o_avatar_icon fa ' + activity.icon + ' bg-' + (activity.state == 'planned'? 'success' : (activity.state == 'today'? 'warning' : 'danger')) + '-full'"
+                            <i t-att-class="'o_avatar_icon ' + activity.icon + ' bg-' + (activity.state == 'planned'? 'success' : (activity.state == 'today'? 'warning' : 'danger')) + '-full'"
```

Comment:
There is a (less than ideal) CSS rule on this icon to center it on its background, and it is not correctly centered at least in community, but this is a long running issue I think.

Reply:
Are you speaking about https://github.com/odoo/odoo/blob/bdb89ee4e99cb4b58ef79f02860034da5fd0d032/addons/mail/static/src/scss/mail_activity.scss#L52-L63 ? Can you show me a screenshot ? Is it *really* odd, or just not pixel perfect ?

Reply:
Not pixel perfect can be really odd for some persons... But objectively its indeed just few pixels off, definitely not breaking the whole interface.

In the context of this PR I would say it is fine to fix later, just reporting.